tests/vmi_hook_sidecar_test.go

```diff
@@ -165,6 +171,127 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 	})
 })
 
+var _ = Describe("[sig-compute]HookQEMUArgsSidecar", decorators.SigCompute, func() {
+
+	var err error
+	var virtClient kubecli.KubevirtClient
+
+	var vmi *v1.VirtualMachineInstance
+
+	BeforeEach(func() {
+		virtClient = kubevirt.Client()
+
+		vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+		vmi.ObjectMeta.Annotations = RenderValidQEMUArgsSidecar(hooksv1alpha1.Version)
+	})
+
+	Describe("[vendor:cnv-qe@redhat.com][level:component] VMI definition", func() {
+		getVMIPod := func(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, bool, error) {
```

Comment:
Can you rely on already existing functions like [GetRunningPodByVirtualMachineInstance](https://github.com/kubevirt/kubevirt/blob/main/tests/utils.go#L321)

Reply:
For this function, we use [UnfinishedVMIPodSelector](https://github.com/kubevirt/kubevirt/blob/main/tests/utils.go#L1373)  as selector so I didn't find a helper function that uses it or accepts ` metav1.ListOptions` as parameter. We could create one as there seems to be enough calls to justify it
```bash
> grep -rniI "UnfinishedVMIPodSelector" tests/ | wc -l
13
```